test/image_processing/lanczos_scaling.cpp

```diff
@@ -0,0 +1,82 @@
+//#include <boost/gil/image_processing/scaling.hpp>
```

Comment:
I assume the above (still non-existent) header is supposed to contain the functions, eventually. What's the reason you haven't used that yet ?